src/sources/util/encoding_config.rs

```diff
@@ -10,11 +10,11 @@ pub struct EncodingConfig {
     /// Takes one of the encoding [label strings](https://encoding.spec.whatwg.org/#concept-encoding-get) defined as
     /// part of the [Encoding Standard](https://encoding.spec.whatwg.org/).
     ///
-    /// When set, the messages are transcoded from the specified encoding to UTF-8, which is the encoding that Vector
-    /// assumes internally for string-like data. You should enable this transcoding operation if you need your data to
-    /// be in UTF-8 for further processing. At the time of transcoding, any malformed sequences (that can’t be mapped to
-    /// UTF-8) will be replaced with the Unicode [REPLACEMENT
-    /// CHARACTER](https://en.wikipedia.org/wiki/Specials_(Unicode_block)#Replacement_character) and warnings will be
+    /// When set, the messages are transcoded from the specified encoding to UTF-8, which is the encoding that is
+    /// assumed internally for string-like data. Enable this transcoding operation if you need your data to
+    /// be in UTF-8 for further processing. At the time of transcoding, any malformed sequences (that can't be mapped to
+    /// UTF-8) is replaced with the Unicode [REPLACEMENT
+    /// CHARACTER](https://en.wikipedia.org/wiki/Specials_(Unicode_block)#Replacement_character) and warnings are
```

Comment:
Noting that we may need to break this link out onto a separate line (I don't remember the name of it, but putting all the links at the bottom of the markdown vs in-line), the existing vector site may not render this in a friendly fashion.